roles/ipaserver/vars/OracleLinux.yml

```diff
@@ -0,0 +1,5 @@
+# defaults file for ipaserver
+# vars/default.yml
+ipaserver_packages: [ "ipa-server", "libselinux-python", "python-six"]
+ipaserver_packages_dns: [ "ipa-server-dns" ]
+ipaserver_packages_adtrust: [ "freeipa-server-trust-ad" ]
```

Comment:
Is the package for the replica different than the package for the server?

Reply:
Thank you for caching that. 
My coworker had informed me that he found it necessary during one of his installs, however I have spent the last day and a half testing this 20+ times in a row with various configurations using OracleLinux 7.7 via the official Vagrant box and the ipaserver, ipareplica and ipaclient roles. I am not able to recreate the requirement for python-six, and I have successfully installed all three (server, replica, and client) after dropping that package.

I have removed that package within a new commit I have pushed, as well as cleaned up the comments and also set the vars to target OracleLinux-7 instead of just OracleLinux. 

Please see my updated changes and let me know if there is any concerns. 
Thank you!

Reply:
Running Oracle Linux 7.7 here (stock, minimal install, adding only the developer_EPEL repo to have ansible available), the name of the package it gives is `ipa-server-trust-ad` and not `freira-server-trust-ad`.

In which version of Oracle Linux is `freipa-server-trust-ad` needed?

Reply:
I am unable to confirm that, and it may have been overlooked, I have removed it and set it to ipa-server-trust-ad instead after confirming this as well.